### PR TITLE
Unixd daemon improvement

### PR DESCRIPTION
**Pull request by MinhPhan8803**
Couple misc fixes for unixd daemon.

- [x] cargo fmt has been run
- [x] cargo clippy has been run
- [x] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

- Removed some `.expects()`
- Move all logs into tracing forest (use the global subscriber).
- Fix cargo.toml (previously couldnt compile in debug build).
- Still no exit code sadly.

### CODE REVIEW

File: unix_integration/Cargo.toml [hunk 1]
```diff
@@ -75,7 +75,7 @@ serde_json.workspace = true
 sketching.workspace = true
 
 toml.workspace = true
-tokio = { workspace = true, features = ["rt", "macros", "sync", "time", "net", "io-util"] }
+tokio = { workspace = true, features = ["rt", "rt-multi-thread", "macros", "sync", "time", "net", "io-util"] }
```
Comment (Firstyear) [inline unix_integration/Cargo.toml:78]: We don't actually want multi-thread here, this was intentional to keep this as single thread.

**Review by Firstyear — CHANGES_REQUESTED** [review]

**Comment by Firstyear** [comment]
Otherwise looks good :)

File: unix_integration/Cargo.toml (hunk 1 above)
Comment (MinhPhan8803) [reply unix_integration/Cargo.toml]: That should be fixed now.

**Review by Firstyear — APPROVED** [review]
Can you also add yourself to contributors.md :)

**Comment by MinhPhan8803** [comment]
> Can you also add yourself to contributors.md :)

Done!

**Comment by Firstyear** [comment]
Thank you!

**Pull request merged**